mod-invoice-storage/examples/voucher_line_collection.sample

```diff
@@ -1,13 +1,24 @@
 {
   "voucherLines": [
-   {
-     "id": "e0d08448-343b-118a-8c2f-4fb50248d672",
-     "externalAccountNumber": "54321098",
-     "voucherLineAmount": 23.45,
-     "fundId": "172364c7-AaAa-A0C2-C12C-e35CeCcbfaF2",
-     "fundDistributions": "efee4b3a-fcd5-40c8-8cf4-0ee3d76ddf4d",
-     "subTransactionId": "7b1f7179-b83f-4eab-a4db-3c600eebe271"
-   }
-  ]
+    {
+      "id": "e0d08448-343b-118a-8c2f-4fb50248d672",
+      "amount": 23.45,
+      "externalAccountNumber": "54321098",
+      "fundDistributions": [
+        {
+          "fundId": "63157e96-0693-426d-b0df-948bacdfdb08",
+          "percentage": 50,
+          "code": "EUHIST",
+          "encumbrance": "5f9bfc74-f6ff-4877-be1f-793f8ce90ade",
+          "invoiceLineId": "e0d08448-343b-118a-8c2f-4fb50248d672"
+        }
+      ],
+      "sourceIds": [
+        "e0d08448-343b-118a-8c2f-4fb50248d672"
```

Comment:
While this is not a technical requirement since these are just examples, I think we should avoid reusing UUIDs as it could lead to confusion.  Here the same UUID is used as the voucherLine.id and a sourceId

Reply:
Voucher line id changed to `7a09dfd7-8588-49b5-bd15-5119414ff0b5`